plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Transform/CaseOfCase.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE LambdaCase    #-}
+{-# LANGUAGE TypeFamilies  #-}
+{-# LANGUAGE TypeOperators #-}
+
+{- | A special case of case-of-case optimisation: transforms
+
+@
+    case ((force ifThenElse) b (constr t) (constr f)) alts
```

Comment:
Should `alts` be `branches`? What do we normally call those BTW? I'd call them `clauses` perhaps, but `branches` seems good too.

Reply:
GHC uses "alt". @michaelpj do we have a standard name?